calcom/cal.com
# feat: Added Google Captcha App

## What does this PR do?
As mentioned in issue #9044 some users were booked by bots hence it was suggested to add the feature of the captcha. So this PR add Google reCaptcha V3 in the form of an app in the cal.com's app store.

Fixes #9294
**Environment**: Staging(main branch) / Production

## Type of change
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How should this be tested?
- Install the `Google reCaptcha V3` app.
- Go to the meet booking page.
- You should see an invisible Google reCaptcha being validated on page load.
- On Successful verification, confirm button would be enabled.

## Screenshots
![Screenshot 2023-06-03 at 12 27 58 AM](https://github.com/calcom/cal.com/assets/44976328/fec2af34-aa01-43fb-8124-348f6f44f4f4)
![Screenshot 2023-06-03 at 12 28 22 AM](https://github.com/calcom/cal.com/assets/44976328/e305bc91-8121-4dc2-9b6e-76fe78ade72f)
